tools/bazel.rc

```diff
@@ -12,6 +12,8 @@ build:opt --copt=-Wframe-larger-than=16384
 build:dbg --compilation_mode=dbg
 
 build:asan --strip=never
+build:asan --copt -Wno-macro-redefined
+build:asan --copt -D_FORTIFY_SOURCE=0
```

Comment:
This is to workaround https://github.com/bazelbuild/bazel/issues/6932

Reply:
- add this explanation to a comment in the file
- does -D_FORTIFY_SOURCE=0 have any impact on the linux asan builds?

Reply:
- Will add a comment
- Linux asan builds work as expected with `-D_FORTIFY_SOURCE=0`